dts/bindings/clock/wch,ch56x-hclk.yaml

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2023 Chen Xingyu <hi@xingrz.me>
+# SPDX-License-Identifier: Apache-2.0
+
+description: WCH CH56x HCLK clock controller
+
+compatible: "wch,ch56x-hclk"
+
+include: [clock-controller.yaml, base.yaml]
+
+properties:
+  clock-frequency:
+    type: int
+    description: |
+      System clock frequency in Hz. Supported values are:
```

Comment:
Can we use MHz instead? The granularity seems to be 1 MHz anyway.

Reply:
I'm not sure. I noticed that another driver (nuvoton,npcx-pcc.yaml) also uses an enum in Hz. I think that most people would naturally assume that a field named `clock-frequency` is expressed in Hz, unless it is explicitly named something like `clock-frequency-mhz`.